pootle/settings/40-apps.conf

```diff
@@ -131,6 +131,10 @@ POOTLE_SIGNUP_ENABLED = True
 # http://django-statici18n.readthedocs.io/en/latest/faq.html#using-a-placeholder-directory
 STATICI18N_ROOT = working_path('static')
 
+# Directory where generated files will be written to. Customized to avoid name
+# clashing with `static/js`.
+STATICI18N_OUTPUT_DIR = 'translations'
```

Comment:
If you read the comment, you'll realize the main point of adding this is to avoid a name clash with `static/js`. If you have a better name which doesn't start with `js`, I'm in.